tenable/io/v3/vm/agent_exclusions/api.py

```diff
@@ -0,0 +1,389 @@
+'''
+Agent Exclusions
+================
+
+The following methods allow for interaction into the Tenable.io
+:devportal:`agent exclusions <agent-exclusions>` API endpoints.
+
+Methods available on ``tio.v3.vm.agent_exclusions``:
+
+.. rst-class:: hide-signature
+.. autoclass:: AgentExclusionsAPI
+    :members:
+'''
+from datetime import datetime
+from typing import Dict
+from uuid import UUID
+
+from marshmallow import EXCLUDE
+from restfly.utils import dict_clean, dict_merge
+
+from tenable.io.v3.base.endpoints.explore import ExploreBaseEndpoint
+from tenable.io.v3.vm.agent_exclusions.schema import AgentExclusionSchema
+
+
+class AgentExclusionsAPI(ExploreBaseEndpoint):
+    '''
+    This class contains methods related to Agent Exclusions APIs
+    '''
+    _path: str = 'api/v3/agents'
+    _conv_json: bool = True
+
+    def create(self,
+               name: str,
+               start_time: str,
+               end_time: str = None,
+               timezone: str = 'Etc/UTC',
```

Comment:
Should this be 'ETC' instead? The casing looks a bit strange, and I'm wondering if it needs to be changed on the API side or not.

Reply:
Also, I would imagine that the default would be UTC alone. Is this even needed if we are providing start/end times in ISO8601?

Reply:
@tomeara currently we are validating timezone using [scan/timezone](https://developer.tenable.com/reference/scans-timezones) API endpoint and I have checked the casing for Etc/UTC,  currently, we are using the same casing that we get from [scan/timezone](https://developer.tenable.com/reference/scans-timezones) API endpoint also the same casing is used in v2 as well.

Also now we are using the start/end time in iso8601 standard format right and so basically timestamp and timezone are the part of the start and end time. so we will remove the timezone parameter from the payload and incorporate these changes in API as well.

Please suggest your thoughts on this?

Reply:
We will take care while IOS8601 while integration testing